sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -67,6 +67,15 @@ __ESIMD_API SurfaceIndex get_surface_index(AccessorTy acc) {
   }
 }
 
+const StringRef SubGroupFuncsResolution::SUB_GROUP_BARRIER = "__builtin_IB_sub_group_barrier";
```

Comment:
REVIEWER: use llvm::StringRef here and in other places

Reply:
PR-OWNER: Fixed in 5786111a9f7ac69349f86aa722a0c5069e25987d

Reply:
PR-OWNER: Sorry, the actual fix is here: 07f5fb9326a95b43969bf8dd691836d9ddfe53c1